app/src/main/res/values/strings.xml

```diff
@@ -154,10 +154,11 @@
     <string name="popup_playing_append">Queued on popup player</string>
     <string name="c3s_url" translatable="false">https://www.c3s.cc/</string>
     <string name="content">Content</string>
-    <string name="show_age_restricted_content_title">Age restricted content</string>
-    <string name="video_is_age_restricted">Show age restricted video. Future changes are possible from the settings.</string>
-    <string name="youtube_restricted_mode_enabled_title">YouTube restricted mode</string>
-    <string name="restricted_video">This video is age restricted.\n\nTurn on \"Age restricted content\" in the settings if you want to see it.</string>
+    <string name="show_age_restricted_content_title">Show age restricted content</string>
```

Comment:
```suggestion
    <string name="show_age_restricted_content_title">18+ content</string>
```
?

Reply:
"Show 18+ content"

Reply:
`Show` should not be removed. And as I said above that setting would block all videos with an age restriction (i.e. even 6+, if they exist), not only 18+ ones